src/gui/viewer.cpp

```diff
@@ -618,7 +643,7 @@ void ImageViewWidget::paintEvent(QPaintEvent* event) {
 	painter.drawImage(target, image_, QRect(0, 0, image_.width(), image_.height()));
 }
 
-void ImageViewWidget::zoom(float amount) {
+void ImageViewWidget::setZoom(float amount) {
```

Comment:
This isn't setting the zoom, it's just zooming in/out by an amount. I think the previous name was fine. `zoomByAmount` could work too or something.

Reply:
reverted to `zoom`